### PR TITLE
refactor(blade): rebrand skeleton

### PR DESCRIPTION
**Pull request by anuraghazra**

### CODE REVIEW

**Review by snitin315 — CHANGES_REQUESTED** [review]
Need to export Skeleton

https://github.com/razorpay/blade/blob/rebranding/master/packages/blade/src/components/index.ts

File: packages/blade/src/components/Skeleton/Skeleton.stories.tsx [hunk 1]
```diff
@@ -141,7 +141,7 @@ const BasicSkeleton = (): React.ReactElement => {
       width="100%"
       padding="spacing.5"
       borderRadius="medium"
-      backgroundColor="surface.background.level2.lowContrast"
+      backgroundColor="surface.background.gray.intense"
```
Comment (snitin315) [inline packages/blade/src/components/Skeleton/Skeleton.stories.tsx:144]: These examples aren't really helpful when we press the show code button in docs to see the implementation.

Screenshot 2024-01-09 at 4 56 49 PM

Ideally, we should display how we have built the shown layout but instead, it just shows `<BasicSkeleton />`.

**Review by snitin315 — APPROVED** [review]
LGTM.

We should improve the docs as per https://github.com/razorpay/blade/pull/1935#discussion_r1445974158 in the near future.

**Review by kamleshchandnani — CHANGES_REQUESTED** [review]

File: packages/blade/src/components/Skeleton/Skeleton.tsx [hunk 2]
```diff
@@ -6,7 +6,6 @@ import { makeAccessible } from '~utils/makeAccessible';
 import { metaAttribute, MetaConstants } from '~utils/metaAttribute';
 
 const Skeleton = ({
-  contrast = 'low',
```
[the remark this answers is not in the document]
Comment (snitin315) [reply packages/blade/src/components/Skeleton/Skeleton.tsx:9]: Not yet, but it's on my radar.

**Comment by kamleshchandnani** [comment]
@anuraghazra rebase

**Comment by anuraghazra** [comment]
done

File: packages/blade/src/components/Skeleton/Skeleton.stories.tsx (hunk 1 above)
Comment (anuraghazra) [reply packages/blade/src/components/Skeleton/Skeleton.stories.tsx:144]: Not possible as of now. Storybook limitation.

**Pull request merged**